src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,185 @@
+@import './variables.scss';
+
+.footer-container{
+	font-size: $fs-16;
+	color:white;
+	text-align: center; 
+	position: relative;
+	width:100%;
+	overflow-x:hidden;
+	background-repeat: no-repeat;
+	background-size: 100% 100%;
+	.footer-bg {
+		width:100%;
+		height:auto;
+		position:absolute;
+		top:0;
+		left:0;
+		z-index:0;
+	}
+	.footer-content {
+		z-index:5;
+		position:relative;
+		width:100%;
+	}
+	.section-title {
+		font-size: $fs-24;
+	}
+	.first{
+		padding-top: 200px;
+	}
+} 
+.top-margin-extra {
+	margin-top:50px;
+}
+
+.footer-text {
+	color:#fff;
+	text-align:left;
+	margin-top:10px;
+	margin-left:100px;
+	margin-right:100px;
+}
+.footer-table{ 
+	margin-left:100px;
+	td {
+		cursor:pointer;
+		color:#ffffff;
+		font-weight:semibold;
+		text-align:left;
+		padding-right:50px;
+		padding-bottom:10px;
+	}
+}
+
+.footer-buttons {
+	margin-top:50px;
+	div {
+		text-align: center;
+		margin:0 auto;
+		.btn-fire {
```

Comment:
I think all buttons style should present in base.scss or a new file `buttons.scss`. Please do that @live-wire